src/Tizen.System/Device/Power.cs

```diff
@@ -154,5 +155,24 @@ public static void ReleaseLock(PowerLock type)
                 throw DeviceExceptionFactory.CreateException(res, "unable to release power lock.");
             }
         }
+        /// <summary>
+        /// Power off the device.
+        /// </summary>
+        /// <exception cref="UnauthorizedAccessException">If the privilege is not set.</exception>
+        /// <exception cref="InvalidOperationException">In case of any system error.</exception>
+        /// <example>
```

Comment:
It must be need a special privilege, but not mentioned which privilege is need,
Please add <privilege> description

Reply:
Applied the request